Front-End/src/Form.js

```diff
@@ -0,0 +1,169 @@
+import React from "react";
+import "./App.css";
+
+// Radio Buttons
+function RadioButtons(props) {
+    let output = [];
+    for (let item in props.items) {
+        output.push(
+            <p key={item}>
+                <input type="radio" id={item} name={props.name} value={item} />
+                <label htmlFor={item}>{props.items[item]}</label>
+            </p>
+        );
+    }
+    return output;
+}
+
+// Number input
+class NumberInput extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { id: props.id, label: props.label, value: 0 };
+        this.setValue = this.setValue.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+        this.increaseValue = this.increaseValue.bind(this);
+        this.decreaseValue = this.decreaseValue.bind(this);
+    }
+
+    setValue(value) {
+        this.setState({ value: Math.abs(parseInt(value)) });
+    }
+
+    handleChange(event) {
+        this.setValue(event.target.value);
+    }
+
+    increaseValue() {
+        this.setState({ value: this.state.value + 1 });
+    }
+
+    decreaseValue() {
+        if (this.state.value > 0)
+            this.setState({ value: this.state.value - 1 });
+    }
+
+    render() {
+        return (
+            <div>
+                <div className="labelleft">
+                    <label htmlFor={this.state.id}>{this.state.label}</label>
+                </div>
+                <div className="buttonright"><input type="button" className="chonk" value="-" onClick={this.decreaseValue} />
+                    <input type="number" value={this.state.value} onChange={this.handleChange} name={this.state.id} min="0" />
+                    <input type="button" className="chonk" value="+" onClick={this.increaseValue} />
+                </div>
+            </div>
+
+
+        );
+    }
+}
+{/* <script>
+// Get the modal
+var modal = document.getElementById('id01');
+
+// When the user clicks anywhere outside of the modal, close it
+window.onclick = function(event) {
+  if (event.target == modal) {
+    modal.style.display = "none";
+  }
+}
+</script> */}
+
+// export { RadioButtons, NumberInput };
+let calendar = document.querySelector('.calendar')
```

Comment:
Are we still using the calendar logic here? If not, it would be smart to remove it before merge

Reply:
fixed.